js/app.js

```diff
@@ -203,7 +203,30 @@ WHERE {
 }
 	`;
 	return all;
-}             
+}
+
+function pickUniversitiesInFloresIsland() {
+  return `
+    #defaultView:Map
+    SELECT ?universityLabel ?long ?lat ?coordinate
```

Comment:
Include `?university` , which creates an i18n'd object on the map.

Do not add `?long ?lat` as properties.  They are the same as `?coordinate`.

Please add more *optional* attributes of universities.  At least `P856` and `P571`.